### DISCUSSION TITLE
Sliding Stretch fails saying "there is not enough memory to paste the selection"

### DISCUSSION THREAD

**Discussion by jserquera** [audacity/audacity#4464] [Q&A]
### Bug description

verison 3.2.4. on Windows

Title says it all. The error is given in Spanish, I'm translating.

### Steps to reproduce

Working with 4 audio tracks.

### Expected behavior

perform a ritardando

### Audacity Version

other

### Operating system

Windows

**Comment by LWinterberg** [comment]
You likely have a clip behind the clip you're trying to timestretch (look out for small black lines). Move these clips further back, or enable "Editing clips can move other clips" in the tracks behavior preferences

**Comment by jserquera** [comment]
That was it: enable "Editing clips can move other clips" 

Thanks!

**Comment by petersampsonaudacity** [comment]
@LWinterberg   -  looks like this one can be closed
